hugegraph-dist/src/main/resources/log4j2.xml

```diff
@@ -7,13 +7,26 @@
             <PatternLayout pattern="%-d{yyyy-MM-dd HH:mm:ss} %-5r [%t] [%-5p] %c %x - %m%n"/>
         </Console>
 
-        <RollingFile name="file" fileName="logs/hugegraph-server.log"
-                     filePattern="logs/$${date:yyyy-MM}/hugegraph-server-%d{yyyy-MM-dd}-%i.log">
+        <!-- Normal server log config -->
+        <RollingRandomAccessFile name="file" fileName="logs/hugegraph-server.log"
+            filePattern="logs/$${date:yyyy-MM}/hugegraph-server-%d{yyyy-MM-dd}-%i.log"
+            immediateFlush="false">
             <ThresholdFilter level="TRACE" onMatch="ACCEPT" onMismatch="DENY"/>
             <PatternLayout pattern="%-d{yyyy-MM-dd HH:mm:ss} %-5r [%t] [%-5p] %c %x - %m%n"/>
             <SizeBasedTriggeringPolicy size="50MB"/>
-        </RollingFile>
+        </RollingRandomAccessFile>
+
+        <!-- Separate audit log, buffer size is 512KB -->
+        <RollingRandomAccessFile name="audit" fileName="logs/audit-hugegraph-server.log"
+            filePattern="logs/$${date:yyyy-MM}/audit-hugegraph-server-%d{yyyy-MM-dd}-%i.log"
+            bufferSize="524288" immediateFlush="false">
```

Comment:
can we use unit like 512kb

Reply:
Seems it can't. There is no unit conversion inside